app/views/works/index.html.erb

```diff
@@ -0,0 +1,89 @@
+<body>
+<main>
+  <h2>List of Works</h2>
+
+  <section class="media-table">
+    <h4>Albums</h4>
+    <table class="table">
+      <thead>
+      <tr>
+        <th width="80">Votes</th>
+        <th>Title</th>
+        <th>Created By</th>
+        <th width="100">Published</th>
+        <th width="100">Upvote</th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @ordered_albums = @albums.sort_by {|album| album.votes.length} %>
+      <% @ordered_albums.reverse.each do |album| %>
+        <tr>
+          <td><%= album.votes.length %></td>
+          <td><%= link_to album.title, work_path(album) %></td>
+          <td><%= album.creator %></td>
+          <td><%= album.publication_year%></td>
+          <td><a class="btn btn-primary" <%= link_to "Upvote", work_votes_path(album.id), method: :post %></a></td>
+        </tr>
+      <%end%>
```

Comment:
Hey look another **very** similar list of works... I wonder how we could dry this up?